packages/theme/src/theme.ts

```diff
@@ -66,6 +66,7 @@ export interface Theme {
     };
 
     status: {
+      default: ContrastColor;
```

Comment:
This should be reflected here:
https://www.figma.com/file/3rlcixpbhfBglNSctUkMys/02.-Text%2C-colors%2C-spacing-and-effects?node-id=2322%3A5000&t=JHF5f17SUiki3dse-1

Also `default` is not a good name for it as it is a reserved word in javascript which sometimes causes some trouble.

Reply:
At the moment didn't create any trouble because we refer to it as a string.

Reply:
The new message box is as of now called `general` in Figma, so I think that is the appropriate name.